packages/account/integration-test/personal-details/personal-details.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from '@playwright/test';
+import { mock_residents_list, mock_states_list, mock_general, mock_loggedIn, setupMocks } from '@deriv/integration';
+
+test.describe('Personal Details', () => {
+    // test('it shows the current name', async ({ page, baseURL }) => {
```

Comment:
@markwylde-deriv we need to check this together, I commented this for now :grimacing: